HealthcareSystem/GraphicalEditor/Models/MapObjectRelated/MapObjectType.cs

```diff
@@ -0,0 +1,43 @@
+﻿using GraphicalEditor.Enumerations;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Media;
+
+namespace GraphicalEditor.Models.MapObjectRelated
+{
+    public class MapObjectType
+    {
+        private MapObjectTypes _mapObjectType;
+
+        public MapObjectType(MapObjectTypes mapObjectType)
+        {
+            this._mapObjectType = mapObjectType;
+        }
```

Comment:
### Comment

For more clear code in C# you should avoid use **this** identifier

![image](https://user-images.githubusercontent.com/45834270/97631292-05535880-1a31-11eb-8604-73dd31e8c2eb.png)

### Proposal 

Remove all this identifiers from code

Reply:
Although using **this** is pretty much redundant in these cases, I really like using it since it helps me to quickly distinguish class fields from other variables, but I will avoid using it in this project if you don't like that behavior.

Reply:
If you are using ' _ ' it say that is local, so i think it is enough to use '_' or 'this' (if you prefer it). But both is not good practice.

Reply:
I don't really mind **this** in constructors, but we should definitely decide on a single practice for everyone and do it consistently. Maybe put up a vote?

Reply:
A single practice for everyone is a good idea. We should up a vote at the next meeting.

Reply:
I agree.